include/nbl/builtin/hlsl/cpp_compat/basic.h

```diff
@@ -11,6 +11,7 @@
 #define NBL_CONSTEXPR constexpr // TODO: rename to NBL_CONSTEXPR_VAR
 #define NBL_CONSTEXPR_FUNC constexpr
 #define NBL_CONSTEXPR_STATIC constexpr static
+#define NBL_CONSTEXPR_INLINE constexpr inline
```

Comment:
`constexpr` implies `inline`

Reply:
you can use `CONSTEXPR_FUNC` instead

Reply:
seems that `NBL_CONSTEXPR_INLINE_FUNC` and `NBL_CONSTEXPR_STATIC_INLINE_FUNC` should be removed

the constexpr inline and constexpr static inline only make sense for variables